content/authors/felix-sosa/_index.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Felix Sosa
```

Comment:
In the UI, there's a "Aliases" -- that's for when you have papers like... F. Sosa, or F. A. Sosa. I would recommend adding those as the site will highlight members of the lab on publications and the like.

Reply:
Done